tests/features/amq/6.2/amq.feature

```diff
@@ -43,4 +43,4 @@ Feature: Openshift AMQ tests
     Then file /tmp/readiness-log should contain stomp port 61613
     Then file /tmp/readiness-log should contain Transport is listening on port 61613
     Then file /tmp/readiness-log should contain stomp+ssl port 61612
-    Then file /tmp/readiness-log should contain Transport is listening on port 61612
\ No newline at end of file
+    Then file /tmp/readiness-log should contain Transport is listening on port 61612
```

Comment:
this is just vi or git or both insisting on a \n being at the end, and adding one.